src/build.rs

```diff
@@ -173,6 +173,14 @@ fn main() {
         println!("cargo:rustc-link-lib=static=ssl");
     }
 
+    if cfg!(feature = "openssl") {
+        #[cfg(feature = "openssl")]
```

Comment:
The order of `libcrypto` and `libssl` below should be swapped. I failed to build quiche with unresolved symbols because the linker command line had `-lcrypto -lssl` in that order. Swapping the two here fixed it.